### PR TITLE
Lambda function in Cpp

### PR DESCRIPTION
**Pull request by Pranshu321**
In this, I have explained how lambda function is defined in code form,

## Related Issue
- #1142

## Proposed Changes
- Lamba Function CPP file
- Lambda functions Readme 

## Additional Info
- In the readme, I deeply explain what the lambda functions how it works, what's usage, and its types of capturing also.
- In the CPP file, it has the executable code of lambda as is defined.

## Checklist

- [x] ✅ My code follows the code style of this project.
- [x] 📝 My change requires a change to the README.
- [x] 🎀 I have updated the README accordingly.
- [x] 👀 I have read the [**CONTRIBUTING**](https://github.com/Lakhankumawat/LearnCPP/blob/main/Contributing.md) document.
- [x] ✨ I have added tests to cover my changes.
- [x] 🚩 All new and existing tests passed.
- [x] 🌟 ed the repo

This is one of the important things of c++11 as it gives the flexibility to define the function on the spot and use it, especially as a comparator function.
So, these topics are covered,
- What's the Lambda function?
- Why use Lambda functions?
- Code and implementation
- Well Documented readme

Lambda function is the one-line function used to do small tasks.

It will add value to your project as it's one of the useful topics of today's time.

#1142 

- [] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to change)

## Output Screenshots
| Screenshot #1      | 
| ----------- | 
| Lambda Function code Output| 
image

### CODE REVIEW

**Comment by Mrjoy832** [comment]
Put ticks on the required fields...not in every one

File: Miscellaneous/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+
+
```
Comment (Mrjoy832) [inline Miscellaneous/README.md:2]: I have changed the readme this time...but from next time for sub title use ###

**Comment by Mrjoy832** [comment]
LGTM🌟

**Comment by Mrjoy832** [comment]
> In this, I have explained how lambda function is defined in code form,
> 
> ## Related Issue
> * [Lambda Functions #1142](https://github.com/Lakhankumawat/LearnCPP/issues/1142)
> 
> ## Proposed Changes
> * Lamba Function CPP file
> * Lambda functions Readme
> 
> ## Additional Info
> * In the readme, I deeply explain what the lambda functions how it works, what's usage, and its types of capturing also.
> * In the CPP file, it has the executable code of lambda as is defined.
> 
> ## Checklist
> * [x]  ✅ My code follows the code style of this project.
> * [x]  📝 My change requires a change to the README.
> * [x]  🎀 I have updated the README accordingly.
> * [x]  👀 I have read the [**CONTRIBUTING**](https://github.com/Lakhankumawat/LearnCPP/blob/main/Contributing.md) document.
> * [x]  ✨ I have added tests to cover my changes.
> * [x]  🚩 All new and existing tests passed.
> * [x]  🌟 ed the repo
> 
> This is one of the important things of c++11 as it gives the flexibility to define the function on the spot and use it, especially as a comparator function. So, these topics are covered,
> 
> * What's the Lambda function?
> * Why use Lambda functions?
> * Code and implementation
> * Well Documented readme
> 
> Lambda function is the one-line function used to do small tasks.
> 
> It will add value to your project as it's one of the useful topics of today's time.
> 
> #1142
> 
> * [x]  Bug fix (non-breaking change which fixes an issue)
> * [x]  New feature (non-breaking change which adds functionality)
> * [x]  Breaking change (fix or feature that would cause existing functionality to change)
> 
> ## Output Screenshots
> Screenshot #1	Screenshot #2
> Title goes here	Title goes here
> Image goes here	Image goes here

Add output screenshot here

**Comment by Pranshu321** [comment]
@Mrjoy832 Sir Screenshot added :) .

File: Miscellaneous/README.md (hunk 1 above)
Comment (Pranshu321) [reply Miscellaneous/README.md:2]: Thank you sir for being so humble, I will take care next time.

**Review by Lakhankumawat — APPROVED** [review]
LGTM 🎉✨

**Comment by Lakhankumawat** [comment]
Keep Contributing like this and i will increase the level of your PR in next PRs

**Comment by Lakhankumawat** [comment]
@all-contributors please add @Pranshu321 code

**Pull request merged**